docs/getting-started/installation-and-upgrade/installation-references/tls-settings.md

```diff
@@ -23,3 +23,82 @@ The default TLS configuration only accepts TLS 1.2 and secure TLS cipher suites.
+
+This setting defaults to `strict` on new installs of Rancher (starting with v2.9.0), but is set to `system-store` for users installing/upgrading from a prior version.
+
+### Preparing for the Setting Change
+
+Each cluster contains a condition in the status field called `AgentTlsStrictCheck`. If its value is set to "True", this indicates the agents for the cluster are ready to operate using the `strict` mode. You can manually inspect each cluster to see if they are ready using the Rancher UI or a kubectl command like the following:
+
+```bash
+## the below command skips ouputs $CLUSTER_NAME,$STATUS for all non-local clusters
+kubectl get cluster.management.cattle.io -o jsonpath='{range .items[?(@.metadata.name!="local")]}{.metadata.name},{.status.conditions[?(@.type=="AgentTlsStrictCheck")].status}{"\n"}{end}'
+```
+
+### Changing the Setting
+
+You can change the setting using the Rancher UI or the `agentTLSMode` [helm chart option](./helm-chart-options.md).
+
+:::note
+
+If you specify the value through the Helm chart, you may only modify the value with helm.
+
+:::
+
+:::warning
+
+Depending on your cert setup, additional action (such as uploading the Certificate Authority which signed your certs) may be required. Review the [installation guide](../install-upgrade-on-a-kubernetes-cluster/install-upgrade-on-a-kubernetes-cluster.md#3-choose-your-ssl-configuration) before changing the setting to see if any additional requirements apply to your setup.
+
+:::
+
+To change the setting's value through the UI, navigate to the "Global Settings" page, and find the `agent-tls-mode` setting near the bottom of the page. When you change the setting through the UI, Rancher will first check that all downstream clusters have the condition `AgentTlsStrictCheck` set to `True` before allowing the request, in order to prevent outages from a certificate mismatch.
+
+
+#### Overriding the Setting Validation Checks
+
+In some cases, you may want to override the check ensuring all agents can accept the new TLS configuration. To do so, you must update the setting using these steps:
+
+:::warning
+
+Rancher checks the status of all downstream clusters to prevent outages. Overriding this check is not recommended, and should be done with great caution.
+
+:::
+
+1. As an admin, generate a kubeconfig for the local cluster. In the below examples, this was saved to the `local_kubeconfig.yaml` file.
+2. Using the following command, retrieve the current setting and save it to the `setting.yaml` file:
+```bash
+kubectl get setting agent-tls-mode -o yaml --kubeconfig=local_kubeconfig.yaml > setting.yaml
+```
+3. Update the `setting.yaml` file, replacing `value` with `strict`. Adding the `cattle.io/force: "true"` annotation will override the cluster condition check, and should only be done with great care:
```

Comment:
```suggestion
3. Update the `setting.yaml` file, replacing `value` with `strict`. Adding the `cattle.io/force: "true"` annotation overrides the cluster condition check, and should only be done with great care:
```

Reply:
Is the last `:` instead of `.` right?

Reply:
Yes, I think in this case it's indicating that the following command is used to perform the action being described in the sentence.